runner/Cargo.toml

```diff
@@ -1,5 +1,3 @@
-[workspace]
```

Comment:
I wonder if the `runner` could be kept as a separate workspace. If we want to publish the workspace on crate.io, it probably does not make sense to include the runner in what we publish. Since the runner is our development tool.

Reply:
The point of the workspace is not for publishing, just for sharing cache, and making it easier for the editor to understand the code. Also when publishing, each crate is completely indepenent of the others, there is no concept of publishing a workspace (that I know of).

Reply:
Great. Thanks for clarifying :)